docs/reference/release-notes/7.10.asciidoc

```diff
@@ -3,6 +3,16 @@
 
 Also see <<breaking-changes-7.10,Breaking changes in 7.10>>.
 
+[discrete]
+=== Known issues
+
+* SQL: in case the WHERE clause of an SQL query contains at least two
+comparisons in a conjunction (AND) of which one is an inequality and the other
+is a not-equals, the later can get ignored. The workaround is to substitute the
+non-equals with a `NOT IN` operator.
+The defect has been fixed in {es} 7.10.1, the best resolution is to upgrade to
+that or a later version. More details can be found in issue: {es-issue}65488[#65488].
```

Comment:
The `inequality` and `not-equal` language confused me and seems to contradict the [SQL docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/sql-operators.html#sql-operators-inequality).

I tried to clarify and add an example:
```suggestion
* SQL: If a `WHERE` clause contains a comparison (`<=`, `<`, `>=`, `>`)
condition and an inequality (`!=`, `<>`) condition joined by `AND`, the SQL
query ignores the inequality condition. For example, the following query ignores
the `foo != 20` condition:
+
[source,sql]
----
SELECT my-field
FROM my-index-000001
WHERE foo > 1 AND foo != 20
----
+
We have fixed this issue in {es} 7.10.1 and later versions. For more details,
see {es-issue}65488[#65488].
```

Reply:
Thanks for the noting the misalignment in therms, @jrodewig. 
I've rephrased the paragraph to better specify the occurring conditions and add back the workaround - which I find relevant and inline with other known issue entries.
I've also left the example out, since we'd then need a corrected example then too and the entry might get large. The GH issue gives examples for both (and links to the resolution too, for whoever would like even more details).
Please have a look and feel free to suggest any further improvements.